### ISSUE TITLE
Package for Almalinux 9

### ISSUE THREAD

**Issue by gwarf** [EGI-Federation/ui-metapackage#4]
Package for CentOS Stream 9.

UMD is not available for Stream 9.
Using EPEL 9, at least those packages are missing:
* gfal2-python
* aria2: likely to be removed, not required?
* ca-policy-egi-core
* condor
* dcache-srmclient
* fts-client
* gfalFS
* ginfo
* lcg-ManageVOTag
* lcg-info
* lcg-infosites
* lcg-tags

**Comment by gwarf** [comment]
[aria2](https://github.com/aria2/aria2) is a command line downloader, not sure it's really required.

**Comment by andrea-manzi** [comment]
trying to build locally on alma9, when installing the package i get 

```
bash-5.1# dnf install /root/workspace/ui-metapackage/build/RPMS/x86_64/ui-6.0.0-1.el9.x86_64.rpm
Last metadata expiration check: 1:23:21 ago on Tue Jun  4 11:04:38 2024.
Error:
 Problem: package ui-6.0.0-1.el9.x86_64 from @commandline requires nordugrid-arc-plugins-needed, but none of the providers can be installed
  - conflicting requests
  - nothing provides libdb_cxx-5.3.so()(64bit) needed by nordugrid-arc-plugins-needed-6.19.0-1.el9.x86_64 from epel
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

indeed 

```
dnf install nordugrid-arc-plugins-needed
Last metadata expiration check: 1:23:50 ago on Tue Jun  4 11:04:38 2024.
Error:
 Problem: cannot install the best candidate for the job
  - nothing provides libdb_cxx-5.3.so()(64bit) needed by nordugrid-arc6-plugins-needed-6.19.0-2.el9.x86_64 from epel
```

**Comment by andrea-manzi** [comment]
need to enable extra repo 

`dnf config-manager --set-enabled crb`

**Comment by andrea-manzi** [comment]
```
 dnf install /root/workspace/ui-metapackage/build/RPMS/x86_64/ui-6.0.0-1.el9.x86_64.rpm
Last metadata expiration check: 0:00:18 ago on Tue Jun  4 12:37:51 2024.
Dependencies resolved.
===============================================================================================================================================================================================================================================
 Package                                                                  Architecture                                Version                                                          Repository                                         Size
===============================================================================================================================================================================================================================================
Installing:
 ui                                                                       x86_64                                      6.0.0-1.el9                                                      @commandline                                       10 k
Upgrading:
 audit-libs                                                               x86_64                                      3.1.2-2.el9                                                      baseos                                            117 k
 gfal2                                                                    x86_64                                      2.22.2-1.el9                                                     epel                                               89 k
 gfal2-all                                                                x86_64                                      2.22.2-1.el9                                                     epel                                              8.5 k
 gfal2-plugin-dcap                                                        x86_64                                      2.22.2-1.el9                                                     epel                                               17 k
 gfal2-plugin-file                                                        x86_64                                      2.22.2-1.el9                                                     epel                                               17 k
 gfal2-plugin-gridftp                                                     x86_64                                      2.22.2-1.el9                                                     epel                                               77 k
 gfal2-plugin-http                                                        x86_64                                      2.22.2-1.el9                                                     epel                                               97 k
 gfal2-plugin-sftp                                                        x86_64                                      2.22.2-1.el9                                                     epel                                               20 k
 gfal2-plugin-srm                                                         x86_64                                      2.22.2-1.el9                                                     epel                                               45 k
 gfal2-plugin-xrootd                                                      x86_64                                      2.22.2-1.el9                                                     epel                                               60 k
 glib2                                                                    x86_64                                      2.68.4-14.el9                                                    baseos                                            2.6 M
 glibc                                                                    x86_64                                      2.34-100.el9_4.2                                                 baseos                                            2.0 M
 glibc-common                                                             x86_64                                      2.34-100.el9_4.2                                                 baseos                                            301 k
 glibc-gconv-extra                                                        x86_64                                      2.34-100.el9_4.2                                                 baseos                                            1.6 M
 glibc-minimal-langpack                                                   x86_64                                      2.34-100.el9_4.2                                                 baseos                                             26 k
 libblkid                                                                 x86_64                                      2.37.4-18.el9                                                    baseos                                            107 k
 libfdisk                                                                 x86_64                                      2.37.4-18.el9                                                    baseos                                            154 k
 libmount                                                                 x86_64                                      2.37.4-18.el9                                                    baseos                                            134 k
 libselinux                                                               x86_64                                      3.6-1.el9                                                        baseos                                             85 k
 libsemanage                                                              x86_64                                      3.6-1.el9                                                        baseos                                            117 k
 libsepol                                                                 x86_64                                      3.6-1.el9                                                        baseos                                            329 k
 libsmartcols                                                             x86_64                                      2.37.4-18.el9                                                    baseos                                             63 k
 libuuid                                                                  x86_64                                      2.37.4-18.el9                                                    baseos                                             27 k
 openldap                                                                 x86_64                                      2.6.6-3.el9                                                      baseos                                            264 k
 openssh                                                                  x86_64                                      8.7p1-38.el9                                                     baseos                                            457 k
 openssh-clients                                                          x86_64                                      8.7p1-38.el9                                                     baseos                                            712 k
 pcre2                                                                    x86_64                                      10.40-5.el9                                                      baseos                                            233 k
 pcre2-syntax                                                             noarch                                      10.40-5.el9                                                      baseos                                            140 k
 python3-rpm                                                              x86_64                                      4.16.1.3-29.el9                                                  baseos                                             64 k
 rpm                                                                      x86_64                                      4.16.1.3-29.el9                                                  baseos                                            486 k
 rpm-build                                                                x86_64                                      4.16.1.3-29.el9                                                  appstream                                          58 k
 rpm-build-libs                                                           x86_64                                      4.16.1.3-29.el9                                                  baseos                                             88 k
 rpm-libs                                                                 x86_64                                      4.16.1.3-29.el9                                                  baseos                                            307 k
 rpm-sign-libs                                                            x86_64                                      4.16.1.3-29.el9                                                  baseos                                             20 k
 util-linux                                                               x86_64                                      2.37.4-18.el9                                                    baseos                                            2.2 M
 util-linux-core                                                          x86_64                                      2.37.4-18.el9                                                    baseos                                            432 k
Installing dependencies:
 aria2                                                                    x86_64                                      1.36.0-4.el9                                                     epel                                              1.2 M
 c-ares                                                                   x86_64                                      1.19.1-1.el9                                                     baseos                                            109 k
 checkpolicy                                                              x86_64                                      3.6-1.el9                                                        appstream                                         351 k
 condor                                                                   x86_64                                      23.7.2-1.el9                                                     htcondor                                          8.0 M
 container-selinux                                                        noarch                                      3:2.229.0-1.el9_3                                                appstream                                          52 k
 dcap                                                                     x86_64                                      2.47.14-1.el9                                                    epel                                               19 k
 dcap-devel                                                               x86_64                                      2.47.14-1.el9                                                    epel                                               11 k
 dcap-tunnel-krb                                                          x86_64                                      2.47.14-1.el9                                                    epel                                               20 k
 dcap-tunnel-ssl                                                          x86_64                                      2.47.14-1.el9                                                    epel                                               13 k
 dcap-tunnel-telnet                                                       x86_64                                      2.47.14-1.el9                                                    epel                                               13 k
 esmtp                                                                    x86_64                                      1.2-19.el9                                                       epel                                               52 k
 fetch-crl                                                                noarch                                      3.0.22-1.el9                                                     epel                                               51 k
 fuse                                                                     x86_64                                      2.9.9-15.el9                                                     baseos                                             79 k
 fuse-libs                                                                x86_64                                      2.9.9-15.el9                                                     baseos                                             96 k
 gfal2-devel                                                              x86_64                                      2.22.2-1.el9                                                     epel                                               35 k
 gfal2-doc                                                                noarch                                      2.22.2-1.el9                                                     epel                                              166 k
 glib2-devel                                                              x86_64                                      2.68.4-14.el9                                                    appstream                                         470 k
 glibc-headers                                                            x86_64                                      2.34-100.el9_4.2                                                 appstream                                         440 k
 glibmm24                                                                 x86_64                                      2.66.1-1.el9                                                     appstream                                         675 k
 globus-gsi-cert-utils-progs                                              noarch                                      10.11-1.el9                                                      epel                                               34 k
 globus-proxy-utils                                                       x86_64                                      7.3-1.el9                                                        epel                                               56 k
 gsi-openssh                                                              x86_64                                      8.7p1-9.el9                                                      epel                                              453 k
 gsi-openssh-clients                                                      x86_64                                      8.7p1-9.el9                                                      epel                                              493 k
 ldns                                                                     x86_64                                      1.7.1-11.el9                                                     appstream                                         160 k
 libattr-devel                                                            x86_64                                      2.5.1-3.el9                                                      appstream                                          25 k
 libblkid-devel                                                           x86_64                                      2.37.4-18.el9                                                    appstream                                          17 k
 libdb-cxx                                                                x86_64                                      5.3.28-53.el9                                                    crb                                               761 k
 libesmtp                                                                 x86_64                                      1.0.6-24.el9                                                     epel                                               66 k
 libffi-devel                                                             x86_64                                      3.4.2-8.el9                                                      appstream                                          28 k
 liblockfile                                                              x86_64                                      1.14-10.el9                                                      baseos                                             28 k
 libmount-devel                                                           x86_64                                      2.37.4-18.el9                                                    appstream                                          18 k
 libselinux-devel                                                         x86_64                                      3.6-1.el9                                                        appstream                                         113 k
 libselinux-utils                                                         x86_64                                      3.6-1.el9                                                        baseos                                            161 k
 libsepol-devel                                                           x86_64                                      3.6-1.el9                                                        appstream                                          39 k
 libsigc++20                                                              x86_64                                      2.10.7-2.el9                                                     appstream                                          38 k
 libxslt                                                                  x86_64                                      1.1.34-9.el9                                                     appstream                                         240 k
 mailcap                                                                  noarch                                      2.1.49-5.el9                                                     baseos                                             32 k
 munge-libs                                                               x86_64                                      0.5.13-13.el9                                                    appstream                                          21 k
 myproxy                                                                  x86_64                                      6.2.16-1.el9                                                     epel                                               68 k
 myproxy-libs                                                             x86_64                                      6.2.16-1.el9                                                     epel                                              131 k
 net-tools                                                                x86_64                                      2.0-0.62.20160912git.el9                                         baseos                                            292 k
 nordugrid-arc                                                            x86_64                                      6.19.0-1.el9                                                     epel                                              1.6 M
 nordugrid-arc-client                                                     x86_64                                      6.19.0-1.el9                                                     epel                                              500 k
 nordugrid-arc-plugins-gfal                                               x86_64                                      6.19.0-1.el9                                                     epel                                               54 k
 nordugrid-arc-plugins-globus                                             x86_64                                      6.19.0-1.el9                                                     epel                                              9.7 k
 nordugrid-arc-plugins-globus-common                                      x86_64                                      6.19.0-1.el9                                                     epel                                               29 k
 nordugrid-arc-plugins-gridftp                                            x86_64                                      6.19.0-1.el9                                                     epel                                               93 k
 nordugrid-arc-plugins-gridftpjob                                         x86_64                                      6.19.0-1.el9                                                     epel                                               69 k
 nordugrid-arc-plugins-lcas-lcmaps                                        x86_64                                      6.19.0-1.el9                                                     epel                                               33 k
 nordugrid-arc-plugins-needed                                             x86_64                                      6.19.0-1.el9                                                     epel                                              1.0 M
 nordugrid-arc-plugins-xrootd                                             x86_64                                      6.19.0-1.el9                                                     epel                                               61 k
 openldap-clients                                                         x86_64                                      2.6.6-3.el9                                                      baseos                                            173 k
 openssh-server                                                           x86_64                                      8.7p1-38.el9                                                     baseos                                            459 k
 pcre-cpp                                                                 x86_64                                      8.44-3.el9.3                                                     appstream                                          26 k
 pcre-devel                                                               x86_64                                      8.44-3.el9.3                                                     appstream                                         470 k
 pcre-utf16                                                               x86_64                                      8.44-3.el9.3                                                     appstream                                         184 k
 pcre-utf32                                                               x86_64                                      8.44-3.el9.3                                                     appstream                                         174 k
 pcre2-devel                                                              x86_64                                      10.40-5.el9                                                      appstream                                         471 k
 pcre2-utf16                                                              x86_64                                      10.40-5.el9                                                      appstream                                         213 k
 pcre2-utf32                                                              x86_64                                      10.40-5.el9                                                      appstream                                         202 k
 pelican                                                                  x86_64                                      7.8.2-1                                                          htcondor                                           16 M
 perl-Compress-Raw-Bzip2                                                  x86_64                                      2.101-5.el9                                                      appstream                                          34 k
 perl-Compress-Raw-Zlib                                                   x86_64                                      2.101-5.el9                                                      appstream                                          60 k
 perl-Data-Dump                                                           noarch                                      1.23-18.el9                                                      appstream                                          32 k
 perl-Date-Manip                                                          noarch                                      6.85-3.el9                                                       appstream                                         1.0 M
 perl-Digest-HMAC                                                         noarch                                      1.03-29.el9                                                      appstream                                          16 k
 perl-Digest-SHA                                                          x86_64                                      1:6.02-461.el9                                                   appstream                                          61 k
 perl-Encode-Locale                                                       noarch                                      1.05-21.el9                                                      appstream                                          19 k
 perl-File-Listing                                                        noarch                                      6.14-4.el9                                                       appstream                                          25 k
 perl-FindBin                                                             noarch                                      1.51-481.el9                                                     appstream                                          12 k
 perl-HTML-Parser                                                         x86_64                                      3.76-3.el9                                                       appstream                                         119 k
 perl-HTML-Tagset                                                         noarch                                      3.20-47.el9                                                      appstream                                          18 k
 perl-HTTP-Cookies                                                        noarch                                      6.10-4.el9                                                       appstream                                          38 k
 perl-HTTP-Date                                                           noarch                                      6.05-7.el9                                                       appstream                                          24 k
 perl-HTTP-Message                                                        noarch                                      6.29-3.el9                                                       appstream                                          96 k
 perl-HTTP-Negotiate                                                      noarch                                      6.01-30.el9                                                      appstream                                          19 k
 perl-I18N-Langinfo                                                       x86_64                                      0.19-481.el9                                                     appstream                                          21 k
 perl-IO-Compress                                                         noarch                                      2.102-4.el9                                                      appstream                                         255 k
 perl-IO-HTML                                                             noarch                                      1.004-4.el9                                                      appstream                                          28 k
 perl-LWP-MediaTypes                                                      noarch                                      6.04-9.el9                                                       appstream                                          33 k
 perl-LWP-Protocol-https                                                  noarch                                      6.10-4.el9                                                       appstream                                          21 k
 perl-NTLM                                                                noarch                                      1.09-30.el9                                                      appstream                                          22 k
 perl-Net-HTTP                                                            noarch                                      6.21-3.el9                                                       appstream                                          40 k
 perl-Sys-Syslog                                                          x86_64                                      0.36-461.el9                                                     appstream                                          47 k
 perl-TimeDate                                                            noarch                                      1:2.33-6.el9                                                     appstream                                          51 k
 perl-Try-Tiny                                                            noarch                                      0.30-13.el9                                                      appstream                                          37 k
 perl-WWW-RobotRules                                                      noarch                                      6.02-30.el9                                                      appstream                                          20 k
 perl-libwww-perl                                                         noarch                                      6.53-4.el9                                                       appstream                                         201 k
 perl-locale                                                              noarch                                      1.09-481.el9                                                     appstream                                          12 k
 policycoreutils                                                          x86_64                                      3.6-2.1.el9                                                      baseos                                            208 k
 python3-audit                                                            x86_64                                      3.1.2-2.el9                                                      appstream                                          82 k
 python3-cffi                                                             x86_64                                      1.14.5-5.el9                                                     baseos                                            241 k
 python3-condor                                                           x86_64                                      23.7.2-1.el9                                                     htcondor                                          912 k
 python3-cryptography                                                     x86_64                                      36.0.1-4.el9                                                     baseos                                            1.1 M
 python3-libselinux                                                       x86_64                                      3.6-1.el9                                                        appstream                                         186 k
 python3-libsemanage                                                      x86_64                                      3.6-1.el9                                                        appstream                                          78 k
 python3-ply                                                              noarch                                      3.11-14.el9                                                      baseos                                            103 k
 python3-policycoreutils                                                  noarch                                      3.6-2.1.el9                                                      appstream                                         2.0 M
 python3-pycparser                                                        noarch                                      2.20-6.el9                                                       baseos                                            124 k
 python3-setools                                                          x86_64                                      4.4.4-1.el9                                                      baseos                                            551 k
 rpm-plugin-selinux                                                       x86_64                                      4.16.1.3-29.el9                                                  baseos                                             16 k
 scitokens-cpp                                                            x86_64                                      1.1.1-1.el9                                                      epel                                              150 k
 selinux-policy                                                           noarch                                      38.1.35-2.el9_4                                                  baseos                                             53 k
 selinux-policy-targeted                                                  noarch                                      38.1.35-2.el9_4                                                  baseos                                            6.5 M
 sysprof-capture-devel                                                    x86_64                                      3.40.1-3.el9                                                     appstream                                          59 k
 xmlsec1                                                                  x86_64                                      1.2.29-9.el9                                                     appstream                                         189 k
 xmlsec1-openssl                                                          x86_64                                      1.2.29-9.el9                                                     appstream                                          90 k
 zlib-devel                                                               x86_64                                      1.2.11-40.el9                                                    appstream                                          44 k
Installing weak dependencies:
 bash-completion                                                          noarch                                      1:2.11-5.el9                                                     baseos                                            291 k
 man-pages                                                                noarch                                      6.04-1.el9                                                       baseos                                            5.7 M
 man-pages-overrides                                                      noarch                                      9.0.0.0-1.el9                                                    appstream                                          16 k
 pelican-osdf-compat                                                      x86_64                                      7.8.2-1                                                          htcondor                                          3.8 k
 perl-Clone                                                               x86_64                                      0.45-6.el9                                                       appstream                                          22 k

Transaction Summary
===============================================================================================================================================================================================================================================
Install  114 Packages
Upgrade   36 Packages
```

**Comment by andrea-manzi** [comment]
Fixed with #7
